src/util/Constants.js

```diff
@@ -34,6 +52,9 @@ const { Error, RangeError } = require('../errors');
  * (or 0 for never)
  * @property {number} [restGlobalRateLimit=0] How many requests to allow sending per second (0 for unlimited, 50 for
  * the standard global limit used by Discord)
+ * @property {string[]|RateLimitQueueFilter} [disableRateLimitQueue] Decides how rate limits (429) should be handled.
+ * If this option is an array containing the route of the request or a function returning true, a {@link RateLimitError}
```

Comment:
I think this could also mention that array elements can be route prefixes too.
e.g. `/channels` matches `/channels/222197033908436994/messages`